rotatescreen/display_linux.py

```diff
@@ -0,0 +1,110 @@
+from Xlib import display
+from Xlib.ext import randr 
+from typing import Dict, List, Tuple
+
+# TODO: Is having these at the global level ok?
+d = display.Display()
+# Create an X display and get the root window
+root = d.screen().root
+# Get the resources for the root window
+res = root.xrandr_get_screen_resources()
+
+
+class Display:
+
+    def __init__(self, output_id, crtc_id):
+        self.output_id = output_id
+        self.crtc_id = crtc_id
+
+    def __repr__(self):
+        return f"<'{self.device_description[0]}' object>"
+
+    def rotate_to(self, degrees: int) -> None:
+        if degrees == 90:
+            rotation_val = randr.Rotate_90
+        elif degrees == 180:
+            rotation_val = randr.Rotate_180
+        elif degrees == 270:
+            rotation_val = randr.Rotate_270
+        elif degrees == 0:
+            rotation_val = randr.Rotate_0
+        else:
+            raise ValueError("Display can only be rotated to 0, 90, 180, or 270 degrees.")
+        
+        # Get the CRTC's current mode and position
+        crtc = d.xrandr_get_crtc_info(self.crtc_id, res.config_timestamp)
+        x, y, mode = crtc.x, crtc.y, crtc.mode
+        
+        # Rotate the CRTC 90 degrees clockwise
```

Comment:
This comment is incorrect, it's not necessarily rotating 90 degrees clockwise, its setting the rotation to `rotation_val`

Reply:
Removed, thanks for the spot!